### PR TITLE
[zh-cn] Rephrase a sentence for fluency

### PR DESCRIPTION
**Pull request by javadoors**
修改“会添被加到”-》“会被添加到”

### CODE REVIEW

**Comment by windsonsea** [comment]
/retitle [zh-cn] Rephrase a sentence for fluency

Could you please use a title in English?

File: content/zh-cn/docs/concepts/workloads/pods/disruptions.md [hunk 1]
```diff
@@ -514,7 +514,7 @@ deleted. In such a situation, after some time, the
 Pod disruption condition will be cleared.
 -->
 Pod 的干扰可能会被中断。控制平面可能会重新尝试继续干扰同一个 Pod，但这没办法保证。
-因此，`DisruptionTarget` 条件可能会添被加到 Pod 上，
+因此，`DisruptionTarget` 条件可能会被添加到 Pod 上，
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/disruptions.md:517]:
```suggestion
因此，`DisruptionTarget` 状况可能会被添加到 Pod 上，
```

**Pull request merged**